catalyst/core/runner.py

```diff
@@ -154,7 +154,8 @@ def _get_experiment_components(
     def _prepare_for_stage(self, stage: str):
         utils.set_global_seed(self.experiment.initial_seed)
         migrating_params = {}
-        migrate = self.experiment.get_state_params(stage).get("migrate", True)
+        stage_state_params = self.experiment.get_state_params(stage)
+        migrate = stage_state_params.get("migrate", True)
```

Comment:
what do you think about `.pop` usage?

Reply:
maybe `migrate` will be important for some callbacks, so let it be?

Reply:
🤔 what about naming then? maybe `migrate_stage=[True/False]`?

Reply:
You mean to rename `migrate` -> `migrate_stage`?

Reply:
@asmekal yup, to make it easier to understand

Reply:
ok, renamed to `migrate_from_previous_stage`